databuilder/models/table_stats.py

```diff
@@ -42,11 +41,7 @@ def __init__(self,
         self.end_epoch = end_epoch
         self.cluster = cluster
         self.stat_name = stat_name
-        try:
-            stat_val = ast.literal_eval(stat_val)
-        except ValueError:
-            stat_val = stat_val
-        self.stat_val = stat_val
+        self.stat_val = str(stat_val)
```

Comment:
if we have list of values for a Table Stat,e.g distinct values or most common values in a column, how do we want handle that? Concat all values to create a single str?

Reply:
Yeah I would think a list would be concatenate to a single string. I think that is how it would be displayed on the front end?

Reply:
@dikshathakur3119 I don't think this model would fit with a list of string(distinct values) . This model is more for displaying scalar data point. For the range list of distinct value, you need to consider implementing differently.